docs: new decision for adding a monitoring tool (#2)
ZcashFoundation/infra-monitoring-kuma: docs/decisions/2024-07-12-monitoring-tool.md

---
# status and date are the only required elements. Feel free to remove the rest.
status: approved
date: 2024-07-12
story: [ZF Engineering update: 2024 Sprint 13 (June 18th - July 1st.)](https://forum.zcashcommunity.com/t/zf-engineering-update-2024-sprint-13-june-18th-july-1st/48210#:~:text=DNSSeeder%20not%20responding%20to%20requests)
---

> # {short title of solved problem and solution}

Reviewer: Missing title here? :)

Author: It's odd GitHub showed you an old version, considering it was fixed a preview commit: [docs(adr): fix title](https://github.com/ZcashFoundation/infra-monitoring-kuma/pull/2/commits/c660da377f2ec9256a461b4825adcf844e9406e7)

## Context & Problem Statement

> The Foundation has multiple production services supporting the network, and we need to monitor them, as we've recently had issues with our DNS Seeder service, and the team realizing after CI failed in Zebra. We need to have a monitoring tool to keep track of the services' health and performance, and be notified proactively when something goes wrong.

Reviewer: ```suggestion
The Foundation runs multiple production services supporting the network, and we need to monitor them, as we've recently had issues with our DNS Seeder service, that have only been noticed once CI failed in Zebra. We need to have a monitoring tool to keep track of the services' health and performance, and be notified proactively when something goes wrong.
```
